tutorial_devops.rst

```diff
@@ -0,0 +1,14 @@
+.. _tutorial_devops:
+
+
+Tutorial - DevOps
```

Comment:
I think this section will be different in how it's treated to the tutorial section. I would suggest to have only one tutorial in the docs that is the previous section and has practical examples and step by step instructions and use an alternative title for this section, as I think it will be more abstract than the tutorial and won't be able to be supported with so many practical examples as the other section is. What about something like:

- Conan for DevOps
- DevOps & Production Guide
- DevOps essentials
- ...

Reply:
The goal is that this section is similar to the other tutorial, with hands-on exercises and explanation. The plan is to add tools that help in using ConanCenter from a fork, and this will be the natural place to explain them. 

But it is also true that this section is also expected to contain sometimes more "abstract" sections with guidelines, etc, so I am fine with changing the title. I like ``Devops & production guide`` a lot, for example.

Reply:
Proposed ``Devops and production``, because it was too long, all the other first-level doc items are just 1 word. I would be happier if we can come up with a 1 or 2 words section name.